global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.100",
+    "version": "9.0.100",
```

Comment:
@justindbaur - Wanted to draw your attention to this. I'm not sure who owns the .Net SDK dependency, but it looks like it needs to update.

Reply:
Yep, and we are currently planning on just jumping to 10. We'll see.